conf/messages

```diff
@@ -158,6 +158,23 @@ employment.employerName.error.noEntry.individual = Enter the name of your employ
 employment.employerName.error.noEntry.agent = Enter the name of your client’s employer
 employment.employerName.error.name.limit = The employer name must be 74 characters or fewer
 
+employment.employmentStartDate.title.individual = When did you start working for your employer?
+employment.employmentStartDate.title.agent = When did your client start working for their employer?
+employment.employmentStartDate.heading.individual = When did you start working at {0}?
+employment.employmentStartDate.heading.agent = When did your client start working at {0}?
+employment.employmentStartDate.forExample = For example, 12 11 2007
+employment.employmentStartDate.error.noEntry.individual = Enter the date your employment started
```

Comment:
It looks like these are unused in the form?

Reply:
Removed the unused imports.
Removed the unused messages.
And the messages thing will be a question for design when i do the demo, like you said its complicated to just do 1 error message when theres three inputs, and surely its more helpful to the user seeing all errors replayed 🤷‍♂️